src/IO.php

```diff
@@ -0,0 +1,170 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Marcosh\LamPHPda;
+
+use Marcosh\LamPHPda\HK\HK1;
+use Marcosh\LamPHPda\Instances\IO\IOApplicative;
+use Marcosh\LamPHPda\Instances\IO\IOApply;
+use Marcosh\LamPHPda\Instances\IO\IOFunctor;
+use Marcosh\LamPHPda\Instances\IO\IOMonad;
+use Marcosh\LamPHPda\Typeclass\Applicative;
+use Marcosh\LamPHPda\Typeclass\Apply;
+use Marcosh\LamPHPda\Typeclass\DefaultInstance\DefaultMonad;
+use Marcosh\LamPHPda\Typeclass\Functor;
+use Marcosh\LamPHPda\Typeclass\Monad;
+use Marcosh\LamPHPda\Brand\IOBrand;
+
+/**
+ * @template A
+ * @implements DefaultMonad<IOBrand, A>
+ */
+final class IO implements DefaultMonad
+{
+    /**
+     * @var callable
```

Comment:
you could be more precise using Psalm. This should be a `callable(): A`

Reply:
Done.